spec/catalog.yaml

```diff
@@ -150,6 +169,19 @@ components:
           example: { "created_at": "1452120468" }
           default: { }
 
+    ListNamespacesResponse:
+      type: object
+      properties:
+        namespaces:
+          description: An array of namespace names in the catalog.
+          type: array
+          uniqueItems: true
+          items:
+            type: array
```

Comment:
I just noticed that this might not be what you mean? Because here you are describing a multi-level namespace where the namespace name is a list of string, but we are just defining namespace name as a simple string.